uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Hook/PageRun/Resources.php

```diff
@@ -21,15 +31,18 @@ public function handle($page) {
       return;
     }
 
-    $this->resources->addScriptFile('uk.co.compucorp.civicrm.tasksassignments',
-        CRM_Core_Config::singleton()->debug
-          ? 'js/src/tasks-assignments.js'
-          : 'js/dist/tasks-assignments.min.js',
-        1010
-      );
+    $isDevEnvironment = CRM_Core_Config::singleton()->debug;
+    $scriptFile = $isDevEnvironment ? 'js/src/tasks-assignments.js' : 'js/dist/tasks-assignments.min.js';
+
+    $this->resources->addScriptFile(self::EXTENSION_KEY,
```

Comment:
Just minor indentation issue here, either put it all on one line or put all the arguments on new lines, like  

```
$this->resources->addScriptFile(
  self::EXTENSION_KEY,
  $scriptFile,
  1010
);

By the way, where is 1010 for the weight coming from? It might be clearer if you made this a variable with a comment